src/openms/source/TRANSFORMATIONS/FEATUREFINDER/FeatureFinderMultiplexAlgorithm.cpp

```diff
@@ -1151,4 +1152,9 @@ namespace OpenMS
   {
     return(consensus_map_);
   }
+  
+  MSExperiment& FeatureFinderMultiplexAlgorithm::getBlacklist()
+  {
+    return(exp_blacklist_);
```

Comment:
you might want to remove these extra braces in return statements:
return exp_blacklist_; is less redundant